### ISSUE TITLE
storybook/notes addon is rendering markdown tables preceeded by numerous <br> tags

### ISSUE THREAD

**Issue by rapheld** [storybooks/storybook#3724]
If you are reporting a bug or requesting support, start here:
### Bug or support request summary
I am using the `storybook/notes` addon. When rendering a table using markdown, I am getting a bunch of `<br>` elements between the proceeding element and the table. This is causing the table to display much farther down the page than intended.

I don't know if this is from something in `Notes` or a bug in `marked`.

### Steps to reproduce
We are using `Notes` to display the `README.md` file associated with a component.

### Please specify which version of Storybook and optionally any affected addons that you're running

- `@storybook/react` 3.4.3
- `@storybook/addon-notes` 3.4.3

```js
// package.json
"@storybook/addon-console": "1.0.0",
"@storybook/addon-info": "3.4.3",
"@storybook/addon-knobs": "3.4.3",
"@storybook/addon-notes": "3.4.3",
"@storybook/addon-options": "3.4.3",
"@storybook/react": "3.4.3",
```

### Affected platforms 

- Google Chrome 66.0.3359.181 (Official Build) (64-bit), OSX 10.12.6
- Seen elsewhere: FF on OSX (don't know versions)

##### Markdown
![storybook_markdown](https://user-images.githubusercontent.com/50293/41065846-cd5fd9de-6994-11e8-9c1c-2654952572ea.png)

##### Rendered page
![storybook_markdown_rendered](https://user-images.githubusercontent.com/50293/41065871-e1fdc7ca-6994-11e8-99bd-62b88fa5c766.png)

##### DOM
![storybook_markdown_dom](https://user-images.githubusercontent.com/50293/41065884-ec6dc872-6994-11e8-8b6d-becf3c7fc4ef.png)

**Comment by Keraito** [comment]
Hey @rapheld, I looked into it and it is caused by Storybook, specifically here: https://github.com/storybooks/storybook/blob/master/addons/notes/src/register.js#L48. The table creates a string of HTML elements, with each of them proceeded by a `\n`. All of them thus get extended with a `<br />`, causing the issue that you're reporting. Unfortunately, there is not negative lookbehind in js, so we will have to find another decent solution.

**Comment by Hypnosphi** [comment]
Released as [`3.4.7`](https://github.com/storybooks/storybook/releases/tag/v3.4.7)
